posthog/api/action.py

```diff
@@ -91,6 +91,17 @@ class Meta:
             "created_by",
         ]
 
+    def update(self, instance, validated_data):
```

Comment:
Hm interesting, though I think this will end up creating a lot of ongoing work for us, as each time we add a field we have to remember to add it here too.

Would it be easier to just `delete` created_by [on this line](https://github.com/PostHog/posthog/pull/1448/files#diff-ec12017d06e7714fe9b2d23fa0a42fddR179)?

Reply:
I don't have a link to the line you sent

but a del request.data['created_by'] somewhere before the update in https://github.com/PostHog/posthog/blob/master/posthog/api/action.py#L167 works as well

Reply:
Yep that sounds good!